pygmt/src/plot3d.py

```diff
@@ -179,12 +181,12 @@ def plot3d(
             )
         extra_arrays.append(kwargs["G"])
         del kwargs["G"]
-    if sizes is not None:
+    if size is not None:
         if kind != "vectors":
             raise GMTInvalidInput(
-                "Can't use arrays for sizes if data is matrix or file."
+                "Can't use arrays for 'size' if data is a matrix or file."
```

Comment:
```suggestion
                "Can't use arrays for 'size' if data is a matrix or a file."
```
Feel free to change it but I'm also fine if you leave it as is.